cartoframes/core/cartodataframe.py

```diff
@@ -24,16 +46,45 @@ def from_features(cls, features, **kwargs):
         return cls(gdf)
 
     def to_carto(self, *args, **kwargs):
+        """
+        Upload a CartoDataFrame to CARTO.
+        Equivalent to :py:meth:`to_carto <cartoframes.io.to_carto>`.
+
+        Examples:
+
+            .. code::
+                cdf.to_carto(if_exists='replace')
+        """
+
         from ..io.carto import to_carto
         return to_carto(self, *args, **kwargs)
 
     def convert(self, index_column=None, geom_column=None, lnglat_columns=None,
                 drop_index=True, drop_geom=True, drop_lnglat=True):
-        # Magic function
+        """
+        Tries to decode the geometry automatically by looking for coordinates in columns.
+
+        Examples:
```

Comment:
An example with `lnglat_columns` can be quite handy. It seems to me that in many cases people working with external files (specially CSVs) will have those coordinates as separate columns.